### PR TITLE
Three Tier on Mobile (V1)

### PR DESCRIPTION
**Pull request by charleycampbell**
## What does this change?
This is the first PR of a few to build out the new banner design. 

It might be helpful to note that as part of this PR:
- new design is only on mobile. There is a placeholder for the tablet and above breakpoint
- all of the data/config is hard coded. There is a piece of work to make it configurable in RRCP 
- wee additional checks, like making the CTA sticky and the banner scrollable

## Why?
This adds in a new design to the banners to support the new choice cards. Based off of this [Figma](https://www.figma.com/design/ONcvFc8ivK6y7SFALDnozu/3-Tier-Banner-Redesign?node-id=970-24579&t=AouJamaMHG8JYaWU-4)

## Screenshots
image

### CODE REVIEW

File: dotcom-rendering/src/components/marketing/banners/designableBanner/DesignableBanner.tsx [hunk 1]
```diff
@@ -626,15 +626,15 @@ const styles = {
 		}
 	`,
 	reminderText: css`
-		${textSans17}
+		${textSans17};
```
Comment (charleycampbell) [inline dotcom-rendering/src/components/marketing/banners/designableBanner/DesignableBanner.tsx:629]: These changes are just a nitpicks that has been annoying me for a while!

File: dotcom-rendering/src/components/marketing/banners/designableBanner/DesignableBannerV2.tsx [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+				});
+			}
+		}
+	}, [iosAppBannerPresent, submitComponentEvent]);
+
+	if (!design) {
+		return <></>;
+	}
+
+	//All config hard coded for the tests - TODO make this configurable in the future
```
Comment (charleycampbell) [inline dotcom-rendering/src/components/marketing/banners/designableBanner/DesignableBannerV2.tsx:67]: Trello card to cover this TODO https://trello.com/c/sLa1GEg7/829-3-tier-banners-wire-in-config-to-tooling

File: dotcom-rendering/src/components/marketing/banners/designableBanner/DesignableBannerV2.tsx [hunk 3]
```diff
@@ -0,0 +1,286 @@
...
+	const templateSettings: BannerTemplateSettings = {
```
Comment (charleycampbell) [inline dotcom-rendering/src/components/marketing/banners/designableBanner/DesignableBannerV2.tsx:68]: Colours in this section will likely need updated as this part of the design isn't signed off (but the tooling might mitigate that!)

File: dotcom-rendering/src/components/marketing/banners/designableBanner/DesignableBannerV2.tsx [hunk 4]
```diff
@@ -0,0 +1,286 @@
...
+		articleCountTextColour: neutral[0],
+		bannerId: 'designable-banner',
+	};
+
+	const showAboveArticleCount =
+		(separateArticleCountSettings?.type === 'above' ||
+			separateArticleCount) &&
+		articleCounts.forTargetedWeeks >= 5;
+
+	return isTabletOrAbove ? (
```
Comment (charleycampbell) [inline dotcom-rendering/src/components/marketing/banners/designableBanner/DesignableBannerV2.tsx:122]: placeholder until the desktop/tablet design is completed

File: dotcom-rendering/src/components/marketing/banners/designableBanner/DesignableBannerV2.tsx [hunk 5]
```diff
@@ -0,0 +1,286 @@
...
+						selectedProduct={threeTierChoiceCardSelectedProduct}
+						setSelectedProduct={
+							setThreeTierChoiceCardSelectedProduct
+						}
+						variantOfChoiceCard={variantOfChoiceCard}
+					/>
+				</div>
+				<div css={styles.linkButtonContainer}>
+					<LinkButton
+						href={
```
Comment (charleycampbell) [inline dotcom-rendering/src/components/marketing/banners/designableBanner/DesignableBannerV2.tsx:180]: I'll re-look into using the Designable CTAs - I couldn't quite get it working initially so this is just temporary

File: dotcom-rendering/src/components/marketing/banners/designableBanner/DesignableBannerV2.tsx (hunk 3 above)
Comment (tomrf1) [reply dotcom-rendering/src/components/marketing/banners/designableBanner/DesignableBannerV2.tsx:70]: 👍 we should use settings from the design tool when we run the AB test

**Review by tomrf1 — APPROVED** [review]
👍

**Pull request merged**